packages/@expo/config-plugins/src/android/Paths.ts

```diff
@@ -18,7 +18,7 @@ export type GradleProjectFile = ProjectFile<'groovy' | 'kt'>;
 
 export function getProjectFilePath(projectRoot: string, name: string): string {
   const filePath = globSync(
```

Comment:
Maybe use the `{ posix: true }` option in glob?

Reply:
We can also use `path.posix.join`

Reply:
This uses both the regex and path.posix.join:

https://github.com/expo/expo/blob/4f7c4948c736de5042ba99c353a46e92668bf0c3/packages/%40expo/metro-config/src/getWatchFolders.ts#L35 

I would assume only one of these is needed. I will test this on my Windows PC later today.